deployment/hasura/metadata/databases/AerieMerlin/tables/public_plan_snapshot.yaml

```diff
@@ -27,6 +42,17 @@ select_permissions:
       columns: '*'
       filter: {}
       allow_aggregations: true
+update_permissions:
+  - role: aerie_admin
+    permission:
+      columns: [description, taken_by]
```

Comment:
What would be the use case for updating the `taken_by` column for a snapshot?

Reply:
Weird username scenario, likely. In general, however, the `aerie_admin` is permitted to update `taken_by` for consistency, as they are allowed to update `owner`/`author`/`requested_by` columns everywhere else. Which, if you're wondering, it can update because it's the admin role and there is not a reason to prevent them from updating that column.